examples/columns2.py

```diff
@@ -0,0 +1,56 @@
+"""
+Builds calendar layout using Columns and Tables.
+Usage:
+python columns2.py [YEAR]
+Example:
+python columns2.py 2021
+"""
+import argparse
+
+import calendar
+from datetime import datetime
+
+from rich.table import Table
+from rich.columns import Columns
+from rich import print
+
+
+def display_calendar(year):
+
+    today = datetime.today()
+    year = int(year)
+    cal = calendar.Calendar()
+
+    tables = []
+    for month in range(1, 13):
+
+        table = Table(title=f"""{calendar.month_name[month]} {year}""", style="green")
+
+        for wd in cal.iterweekdays():
+            table.add_column("{:.3}".format(calendar.day_name[wd]))
+
+        month_days = cal.monthdayscalendar(year, month)
+        for weekdays in month_days:
+            days = []
+            for day in weekdays:
+                today_tuple = today.day, today.month, today.year
+
+                if day and (day, month, year) == today_tuple:
+                    _day = "[bold underline red]%s[/]" % day
+                else:
+                    _day = "%s" % ("" if day == 0 else day)
```

Comment:
I don't think you need to interpolate at all here.

How bout this?

```python
_day = "" if day == 0 else str(day)
```

Reply:
good find.